content/docs/setup/kubernetes/minimal-install/index.md

```diff
@@ -69,21 +61,23 @@ to manage the lifecycle of Istio.
     $ helm init --service-account tiller
     {{< /text >}}
 
-1. Install Istio:
+1. Install the `istio-init` chart to bootstrap all the Istio's CRDs:
+
+    {{< text bash >}}
+    $ helm install install/kubernetes/helm/istio-init --name istio-init --namespace istio-system
+    {{< /text >}}
+
+1. Verify all the Istio's CRDs have been committed in the Kube api-server by checking the number of CRD with specified suffix. Run the following command and check the result, totally `57` CRDs should be created for Istio:
```

Comment:
Mind the line length. The text is a bit hard to follow due to all the different tenses present. Try simplifying the text to make it easier to understand:
```suggestion
1. To verify all Istio's CRDs were committed in the Kube api-server, check the number of CRDs with the specified suffix with the following command. Verify that the number of total CRDs created was `57` for Istio:
```

Reply:
I would change `Kube api-server to "Kubernetes api-server`.  I am also not clear that 57 is accurate.  On a fresh cluster, I get 59:

kubectl get crds | grep 'istio.io\|certmanager.k8s.io' | wc -l

Reply:
@sdake Check CRD number again with latest `release-1.1`:
49 CRDs in https://github.com/istio/istio/blob/release-1.1/install/kubernetes/helm/istio-init/files/crd-10.yaml
4 CRDs in https://github.com/istio/istio/blob/release-1.1/install/kubernetes/helm/istio-init/files/crd-11.yaml
3 CRDs in https://github.com/istio/istio/blob/release-1.1/install/kubernetes/helm/istio-init/files/crd-certmanager-10.yaml